### PR TITLE
docs(examples): add action-items widget examples

### PR DESCRIPTION
**Pull request by apetro**
Add several `action-items` widget examples to localhost demonstrating this widget type across variety of states. Demonstrates some opportunities for improvement for pursuit in subsequent Pull Requests.

After:

action-items-in-localhost-examples

Refactors how we tell `eslint` what to ignore, since there are now 3 paths beyond those in `.gitignore`. https://github.com/eslint/eslint/issues/9794 might eventually allow telling `eslint` to ignore both files in `.gitignore` and files in `.eslintignore`; in the meantime introduces `.eslintignore` which declares a superset (via copy-and-paste 😞 ) of paths in `.gitignore`. 

----

PR considerations checklist:

- [x] Updates `CHANGELOG.md` to reflect this PR's change.
- [x] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

**Comment by apetro** [comment]
CI failure appears to be commit message linting not liking `chore: .eslintignore quantity_ill-formed.json` because of the dot prefix on `.eslitignore`. This'll be moot when this squashes onto master with a message more like `docs(examples): add action-items widget examples`.

File: package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
     "lint-all": "npm run lint-js && npm run lint-md && npm run lint-commit && npm run lint-css",
     "lint-staged": "lint-staged",
     "lint-css": "stylelint --syntax less components/css/**/*.less",
-    "dev-lint-js": "onchange components/** -i -- eslint --quiet --ext js --ext json --ext md --format html --output-file static/target/lint-out.html . --ignore-path .gitignore --ignore-pattern components/js/ga.js",
```
Comment (davidmsibley) [inline package.json]: I get the motivation behind pulling these patterns out into a file, but if the alternative is maintaining duplicated `.gitignore` and `.eslintignore`, I'd rather just add one more pattern here.

**Comment by apetro** [comment]
Superseded by #835 , same change implemented in cleaner commits.

File: package.json (hunk 1 above)
Comment (apetro) [reply package.json]: #835 goes with adding another `--ignore-pattern` rather than introducing a `.eslintignore`.

**Pull request closed without merging**